demisto_sdk/commands/content_graph/strict_objects/dashboard.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Optional
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.strict_objects.base_strict_model import (
+    BaseOptionalVersionJson,
+)
+from demisto_sdk.commands.content_graph.strict_objects.common import (
+    DESCRIPTION_DYNAMIC_MODEL,
+    ID_DYNAMIC_MODEL,
+    NAME_DYNAMIC_MODEL,
+    BaseStrictModel,
+    create_dynamic_model,
+    create_model,
+)
+
+
+class _DashboardLayout(BaseStrictModel):
+    force_range: bool = Field(alias="forceRange")
+    x: int
+    y: int
+    h: int
+    w: int
+    i: str
+    widget: dict = Field(default_factory=dict)
```

Comment:
can we model the widget dictionary too? based on what we have in the repo

Reply:
Done